### PR TITLE
Add map component to Main screen.

### PR DESCRIPTION
**Pull request by nsantos16**
### Description
As a user I should be able to see a map so I can easily identify targets in the future

### Preview
iosmap

### CODE REVIEW

File: src/components/common/TextInputExtend/index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ class TextInputExtend extends React.Component {
     return <TextInput
       {...rest}
       onChange={this.handleChange}
-      style={placeholder ? [style, placeholderStyle] : style}
+      style={[style, placeholder ? placeholderStyle : { color: '#000' }]}
```
Comment (agustito37) [inline src/components/common/TextInputExtend/index.js]: Alo! This style should be placed in the styles files.

File: src/containers/MainScreen/styles.js [hunk 2]
```diff
@@ -3,8 +3,9 @@ import { StyleSheet } from 'react-native';
 const styles = StyleSheet.create({
   container: {
     flex: 1,
+    flexDirection: 'row',
     justifyContent: 'center',
-    alignItems: 'center',
+    alignItems: 'flex-end',
     backgroundColor: '#F5FCFF',
```
Comment (agustito37) [inline src/containers/MainScreen/styles.js]: Remember to save the colors inside a constant file to reuse them later.

File: src/components/main/Map/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { reduxForm } from 'redux-form/immutable';
```
Comment (felipegarcia92) [inline src/components/main/Map/index.js]: Why the form here?

Comment (nsantos16) [reply src/components/main/Map/index.js]: It's not necessary right now, but in the future  I suppose the position on the map should be in the store.

**Pull request merged**